src/main/java/com/fakng/fakngagrgtr/parser/stripe/StripeParser.java

```diff
@@ -0,0 +1,132 @@
+    }
+
+    private DomNodeList<DomNode> getLinksFromPage(HtmlPage page) {
+        return page.querySelectorAll(".JobsListings__link");
+    }
+
+    private List<Vacancy> fillVacancy2(DomNodeList<DomNode> links) {
+        List<Vacancy> vacancies = new ArrayList<>();
+        links.forEach(link ->
+                vacancies.add(fillVacancyDTO(link.getAttributes().getNamedItem("href").getNodeValue().substring(13)))
+        );
+
+        return vacancies;
+    }
+
+    @SneakyThrows
+    private Vacancy fillVacancyDTO(String link) {
+        VacancyDTO vacancyDTO = new VacancyDTO();
+
+        HtmlPage page = downloadPage(VACANCY_URL + link);
+        DomNodeList<DomNode> description = page.querySelectorAll(".ArticleMarkdown");
+        DomNodeList<DomNode> jobElements = page.querySelectorAll(".JobDetailCardProperty__title");
+        HtmlHeading1 title = (HtmlHeading1) page.getByXPath(VACANCY_TITLE_XPATH).get(0);
+
+        vacancyDTO.setDescription("");
+
+        description.get(0).getChildren().forEach(
+                iter -> vacancyDTO.setDescription(vacancyDTO.getDescription() + iter.asNormalizedText())
+        );
+
+        vacancyDTO.setLink(link);
+        vacancyDTO.setOfficeLocation(List.of(jobElements.get(0).getNextElementSibling().getTextContent().replace(" or ", "").split(", ")));
+        vacancyDTO.setJobType(jobElements.get(jobElements.size() - 1).getNextElementSibling().getTextContent());
+        vacancyDTO.setTeam(jobElements.get(jobElements.size() - 2).getNextElementSibling().getTextContent());
+        vacancyDTO.setRoles(title.getTextContent());
+
+        if (jobElements.size() == 4) {
+            vacancyDTO.setHasRemote(true);
+            vacancyDTO.setRemoteLocation(List.of(jobElements.get(1).getNextElementSibling().getTextContent().replace(" or ", "").split(", ")));
+        }
+
+        System.out.println(vacancyDTO);
+
+        return fillVacancy(vacancyDTO);
+    }
+
+    private Vacancy fillVacancy(VacancyDTO vacancyDTO) {
+        Vacancy vacancy = new Vacancy();
+
+        List<String> locations = new ArrayList<>();
+        String description = vacancyDTO.getDescription() +
+                (vacancyDTO.getHasRemote() ? "Has Remote: true" : "Has Remote: false");
+
+        if (vacancyDTO.getRemoteLocation() != null) {
+            locations.addAll(vacancyDTO.getRemoteLocation());
+        }
+        locations.addAll(vacancyDTO.getOfficeLocation());
+
+        vacancy.setTitle(vacancyDTO.getRoles());
+        vacancy.setUrl("https://stripe.com/jobs/listing" + vacancyDTO.getLink());
+        vacancy.setDescription(description);
+        vacancy.setCompany(company);
+        processLocations(vacancy, locations);
+
+        return vacancy;
+    }
+
+    private void processLocations(Vacancy vacancy, List<String> locations) {
+        locations.forEach(location -> {
+            vacancy.addLocation(locationProcessor.processLocation(
```

Comment:
So this method accepts city and country code. You pass for both params just location. It just breaks the structure we have in DB. First param - city, second - country code.